### DISCUSSION TITLE
DNS SERVER

### DISCUSSION THREAD

**Discussion by palsachennaiah** [learndevopsonline/learndevopsonline#1567] [Q&A]
1) Three days back I created domain (hostinger) DNS server have not shown still blue tick
 only 6 countries has blue ticks.

**Comment by sreedharm07** [comment]
actually it will be available in 24 hrs in all locations. but in your case its not. 
so, delete the hosted zone in aws and create again.
after creating new hosted zone change name servers with newly created hostedzone in hostinger.

**Reply by palsachennaiah** [reply]
How would i delete hosted zone

**Reply by sreedharm07** [reply]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/7d8e743e-53a9-489d-b67c-a6f26589e747)

**Comment by palsachennaiah** [comment]
![Screenshot 2023-11-04 094344](https://github.com/learndevopsonline/learndevopsonline/assets/148023299/9a9e3780-2e03-4487-8d9e-a4a6484d42d8)

**Comment by palsachennaiah** [comment]
![Screenshot 2023-11-04 094720](https://github.com/learndevopsonline/learndevopsonline/assets/148023299/7c66c476-58d8-4cbc-9de8-69ceb0478e84)

**Reply by sreedharm07** [reply]
1 st delete the 3 rd record (www.cdev...) 
 then delete hosted zone

**Comment by palsachennaiah** [comment]
task not completed

**Reply by sreedharm07** [reply]
give screenshot please

**Comment by palsachennaiah** [comment]
Thank you problem solve

**Comment by vikram9491** [comment]
> Thank you problem solve
> […](#)
> On Sat, 4 Nov 2023 at 10:19, Palsa Chennaiah ***@***.***> wrote: On Sat, 4 Nov 2023 at 10:10, Palsa Chennaiah ***@***.***> wrote: > > > On Sat, 4 Nov 2023 at 10:05, sreedharm07 ***@***.***> > wrote: > >> give screenshot please >> >> — >> Reply to this email directly, view it on GitHub >> <[#1567 (reply in thread)](https://github.com/learndevopsonline/learndevopsonline/discussions/1567#discussioncomment-7472462)>, >> or unsubscribe >> <https://github.com/notifications/unsubscribe-auth/BDJKQA6IM3QHSF4AWCSVVWDYCXAXPAVCNFSM6AAAAAA65I6IB6VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TINZSGQ3DE> >> . >> You are receiving this because you authored the thread.Message ID: >> <learndevopsonline/learndevopsonline/repo-discussions/1567/comments/7472462 >> @github.com> >> >

hi whether your domain name checked in all locations ...? same issue i am getting
